store/task/actions.js

```diff
@@ -0,0 +1,179 @@
+import {
+  ACTION_ADD_TASK,
+  ACTION_UPDATE_TASK,
+  ACTION_DELETE_TASK,
+  ACTION_GET_ALL_TASKS,
+  ACTION_GET_COMPLETED_TASKS,
+  ACTION_UPDATE_TASK_API_STATUS
+} from './action_types'
+import {
+  TASK_API_RESET,
+  TASK_API_INITIATED,
+  TASK_API_COMPLETED,
+  TASK_API_FAILED
+} from './api_status'
+import axios from 'axios'
+
+import { axiosSetup } from '../../lib/axios-settings'
```

Comment:
Initiate the `axiosSetup` in `_app.js`component

Reply:
Updated